Pipfile

```diff
@@ -4,18 +4,18 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pre-commit = "~=1.13"
+pre-commit = "~=1.21"
```

Comment:
it would be interesting to know if Python has something similar to NodeJS's Greenkeeper, which monitors dependency versions

Reply:
@mykola-mokhnach 

`dependabot` can work.
https://github.com/ki4070ma/change-passwd-func/commits/master

Reply:
Added
https://app.dependabot.com/accounts/appium/projects/138937

Reply:
Thanks @KazuCocoa